src/filetypes.c

```diff
@@ -604,6 +605,7 @@ static GeanyFiletype *find_shebang(const gchar *utf8_filename, const gchar *line
 			{ "sh",		GEANY_FILETYPES_SH },
 			{ "bash",	GEANY_FILETYPES_SH },
 			{ "dash",	GEANY_FILETYPES_SH },
+			{ "perl6",	GEANY_FILETYPES_RAKU },
```

Comment:
Given the run command is `raku "%f"`, shouldn't `raku` be included in this list as well?

Reply:
Yeah, definitely, I've just added that.